### ISSUE TITLE
Laptop does not resume (hangs), ACPI error (may be related)

### ISSUE THREAD

**Issue by srdjanpavlovic** [linuxdeepin/developer-center#3430]
### SUMMARY | 问题概要

HP Laptop throws ACPI Error AE_NOT_FOUND

### STEPS TO REPRODUCE | 复现步骤

1. Install Deepin 20.7 on HP laptop
2. Boot (System boots, but ACPI Error is shown before boot)
3. Go to sleep / hibernate
4. Try to resume

### OBSERVED RESULT | 观察到的结果

Laptop does not resume from hibernation (System hangs, only hard reset possible).

### EXPECTED RESULT | 期望的结果

Expected - no ACPI errors and resume works (does not hang and require hard reset)

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

Deepin 20.7, default instalation.

 HP Laptop, model: 15s-eq1042nm, prod ID 1U3F1EA#AKQ

### ADDITIONAL INFORMATION | 额外补充

Exact Error:

 ACPI Error: AE_NOT_FOUND, While resolving a name reference package element - \_PR_.P000 (20211217/dspkginit-440)

**Comment by srdjanpavlovic** [comment]
If any of system logs are helpful, please ask and I will attach it here.

**Comment by tsic404** [comment]
ACPI error seems a firmware bug. Try to add `acpi=off` in kernel command line by editing /etc/default/grub, after `update-grub` and `reboot`, can it resume from a hibernation?

**Comment by srdjanpavlovic** [comment]
MS Windows hibernates fine, so it is something in Linux Kernel that handles ACPI on this laptop. acpi=off may be a workaround, but I would rather like if someone who knows why this error is happening debug this, so it works on Linux as it works on Windows.

**Comment by tsic404** [comment]
Most bios are adopted to windows and have little support for linux, you can try to upgrade bios from your laptop vendor support.

**Comment by srdjanpavlovic** [comment]
Yeah, I know that, I do have most recent BIOS on Laptop. I will try acpi=off as a workaround, but maybe someone can fix this on Linux, too.
